combirepo/config_parser.py

```diff
@@ -45,7 +45,7 @@ def initialize_config(config_file_path):
     global default_path
```

Comment:
BTW, above link (help_url = "http://confluence.rnd.samsung.ru/display/TC/Config+file+format") is not visible from outside. But this is not an issue of this patch.

Reply:
Anyway we should remove link from source.
Maybe it'll be better to move this doc into additional man page?

Reply:
...or into wiki of this project